sdk/keyvault/azure-keyvault-keys/tests/_test_case.py

```diff
@@ -2,18 +2,60 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
+import asyncio
```

Comment:
This will raise an ImportError on 2.7. If you want to combine a/sync in one module, you need to do [something like this](https://github.com/Azure/azure-sdk-for-python/blob/5ec46673404af7aa30fe6e4161b7687d1521bdd2/tools/azure-devtools/src/azure_devtools/scenario_tests/preparers.py#L146-L155).